bigquery/bigquery_create_reservation/main.tf

```
@@ -27,3 +28,10 @@ resource "google_bigquery_reservation" "default" {
   }
 }
 # [END bigquery_create_reservation]
+
+resource "google_bigquery_reservation_assignment" "default" {
+  assignee    = "projects/my-project"
```

This value should be based on a variable, not hardcoded to a value. This config should work: 

```suggestion
data "google_project" "project" {}

resource "google_bigquery_reservation_assignment" "default" {
  assignee    = "projects/${data.google_project.project.project_id}"
```

---

I'm fine this change if using variables is now the guidance.